src/cljs/athens/views/blocks/textarea_keydown.cljs

```diff
@@ -531,11 +531,13 @@
 
 (defn handle-escape
   "BUG: escape is fired 24 times for some reason."
-  [uid e]
+  [e uid {:keys [esc-handler] :as _state-hooks}]
   (.. e preventDefault)
-  (if @(rf/subscribe [::inline-search.subs/type uid])
-    (rf/dispatch [::inline-search.events/close! uid])
-    (dispatch [:editing/uid nil])))
+  (if (fn? esc-handler)
+    (esc-handler e uid)
+    (if @(rf/subscribe [::inline-search.subs/type uid])
+      (rf/dispatch [::inline-search.events/close! uid])
+      (dispatch [:editing/uid nil]))))
```

Comment:
I added an esc-handler for some of task title editors on kanban and table views. In those cases, esc should cancel the title edit, and blurring should save. Wanted to bring this to your attention @neotyk to double-check this is an ok way to do things